Dockerfile.rie

```diff
@@ -0,0 +1,16 @@
+FROM public.ecr.aws/lambda/ruby:3.2
+
+# Copy source code and build gem in container
+COPY . ${LAMBDA_TASK_ROOT}/
+WORKDIR ${LAMBDA_TASK_ROOT}
+
+# Build and install the RIC gem
```

Comment:
should we uninstall the previous gem just to be sure we're starting from a clean state?